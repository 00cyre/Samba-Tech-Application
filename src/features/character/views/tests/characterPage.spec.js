import React from 'react';
import CharacterPage from '../characterPage';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
describe('Component <CharacterPage/>', function () {
    let component;
    let getCharacter = jest.fn();
    let Characters = [{ name: "Walter White" }];
    beforeEach(() => {
        component = shallow(
            <CharacterPage
                getCharacter={getCharacter}
                Characters={Characters}
            />
        );
    });

    it('Should Render <CharacterPage/> ', () => {
        expect(component).toMatchSnapshot();
    });

});
