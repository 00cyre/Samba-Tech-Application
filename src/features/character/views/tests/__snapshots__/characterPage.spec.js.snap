// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <CharacterPage/> Should Render <CharacterPage/>  1`] = `
<CharacterPage
  Characters={
    Array [
      Object {
        "name": "Walter White",
      },
    ]
  }
  classes={
    Object {
      "components": "CharacterPage-components-1",
      "content": "CharacterPage-content-2",
    }
  }
  getCharacter={[MockFunction]}
/>
`;
